packages/ag-charts-community/src/util/properties.ts

```diff
@@ -72,6 +73,58 @@ export class PropertiesArray<T extends BaseProperties> extends Array {
     }
 }
 
+export class OptionalPropertiesArray<T extends BaseProperties> {
```

Comment:
Maybe sync with @iMoses about the use-case for this?

Reply:
Resolved. We've decided to use an empty `PropertiesArray` instance instead.